reference/prefix_regex.md

```diff
@@ -0,0 +1,23 @@
+# Prefix regex
+
+## Using `prefix` and `prefix_regex`
+
+When the `prefix_regex` attribute is set to `true`, ambassador configures a regex route instead of a prefix route in envoy. **This means the entire path must match the regex specified, not only the prefix.** 
```

Comment:
```suggestion
When the `prefix_regex` attribute is set to `true`, ambassador configures a [regex route](https://www.envoyproxy.io/docs/envoy/v1.5.0/api-v1/route_config/route#route) instead of a prefix route in envoy. **This means the entire path must match the regex specified, not only the prefix.** 
```